examples/04_manipulating_images/plot_extract_regions_labels_image.py

```diff
@@ -0,0 +1,118 @@
+"""
+Regions labeling and extraction using Yeo atlas
+=============================================================
+
+This example shows how to use :class:nilearn.regions.extract_regions_labels_img
+to assigns each separated region of the atlas a unique label.
+
+We use the Yeo atlas as an example for labeling regiones,
+:func:nilearn.datasets.fetch_atlas_yeo_2011
+
+"""
+
+################################################################################
+# Fetching the Yeo atlas
+from nilearn import datasets
+
+atlas_yeo_2011 = datasets.fetch_atlas_yeo_2011()
+atlas_yeo = atlas_yeo_2011.thick_7
+
+################################################################################
+# Labeling the atlas regions
+from nilearn.regions import extract_regions_labels_img
+region_labels = extract_regions_labels_img(atlas_yeo,
+                                           min_size=None,
+                                           connect_diag=True)
+
+################################################################################
+# Counting regions and potting them
+import nibabel as nb
+import numpy as np
+atlas_yeo_img = nb.load(atlas_yeo)
+atlas_yeo_data = atlas_yeo_img.get_data()
+region_labels_data = region_labels.get_data()
+
+print("The regions of Yeo atlas: ", np.unique(atlas_yeo_data))
+print("The labeled regions of Yeo atlas: ", np.unique(region_labels_data))
+
+# Let's plot them
+# The original Yeo atlas has 7 regions, that is indicated in the colorbar.
+# The colorbar also shows the corespondence to the color and the cluster
+from nilearn import plotting
+
+# the original Yeo atlases
+pl_yeo = plotting.plot_roi(atlas_yeo, title='Original Yeo atlas',
+                           cut_coords=(8,-4,9),
+                           cmap = 'gist_rainbow', colorbar=True)
+plotting.show()
```

Comment:
plotting.show() can be removed. We can use one plotting.show() at the end to show them. One at line 58 should be enough.